README.md

```diff
@@ -1,3 +1,20 @@
-# Welcome!
+LibreClinica
+============
 
-LibreClinica is the community driven successor of OpenClinica. It is an open source clinical trial software for Electronic Data Capture (EDC) Clinical Data Management (CDM). https://libreclinica.org
+_the real open source electronic data capture (EDC) for clinical studies._
+
+[LibreClinica](https://libreclinica.org) is the community driven successor of OpenClinica. It is an open source clinical trial software for Electronic Data Capture (EDC) and Clinical Data Management (CDM).
+
+### Getting Started
+
+Check out [LibreClinica Documentation](https://libreclinica.org/documentation) to learn how to install, validate and use the software.
+
+### Development
+
+In case of interest in contributing into the codebase, check out [LibreClinica Developer Documentation](https://libreclinica-docs.readthedocs.io) to learn how to set up the development workstation and understand the branching strategy that is used to control the contribution workflows. 
+
+### Contributions
+                          
+LibreClinica is open source project that values input from contributors:
+
+* Thomas Hillger, University Hospital RWTH Aachen
```

Comment:
We should ask Thomas Hillger, whether other colleagues (Julia Bley) should be also mentioned.

Reply:
sure, if there have been more people involved, just let me know and I update readme